src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -423,8 +423,8 @@ def _normalize_properties(self, user, subscriptions, is_service_principal, cert_
 
             if cert_sn_issuer_auth:
                 consolidated[-1][_USER_ENTITY][_SERVICE_PRINCIPAL_CERT_SN_ISSUER_AUTH] = True
-            if user_assigned_identity_id:
-                consolidated[-1][_USER_ENTITY][_ASSIGNED_IDENTITY_INFO] = user_assigned_identity_id
+            if assigned_identity_info:
+                consolidated[-1][_USER_ENTITY][_ASSIGNED_IDENTITY_INFO] = assigned_identity_info
```

Comment:
`consolidated[-1][_USER_ENTITY][_ASSIGNED_IDENTITY_INFO]` can also be `MSI` to represent system-assigned managed identity, so the name `user_assigned_identity_id` is not accurate. Aligning it with the property name `assignedIdentityInfo`.